frontend/dashboard/src/layout/dropdowns/dropdownProfileOptions.vue

```diff
@@ -0,0 +1,51 @@
+<script setup lang="ts">
+import { IconLogout } from '@tabler/icons-vue';
+import { DropdownOption, NAvatar, NButton, NDropdown, NSpin } from 'naive-ui';
+import { useI18n } from 'vue-i18n';
+
+import { useLogout } from '@/api';
+import { renderIcon } from '@/helpers';
+
+
+defineProps<{
+	isProfileLoading: boolean,
+	avatar?:  string,
+}>();
```

Comment:
Isn't better to get profile via tanstack hook, so in profile you can get `isLoading` and `avatar`?